code/modules/power/generators/turbine.dm

```diff
@@ -203,8 +246,7 @@
 // These are crucial to working of a turbine - the stats modify the power output. TurbGenQ modifies how much raw energy can you get from
 // rpms, TurbGenG modifies the shape of the curve - the lower the value the less straight the curve is.
 
-#define TURBPRES 9000000
-#define TURBGENQ 100000
+#define TURBGENQ 500000
 #define TURBGENG 0.5
```

Comment:
would you rename these defines and give them documentation? `TURBGENQ` tells me literally nothing

Reply:
Frankly, I have no idea what they meant either. Its what they were always called. They have been renamed in latest to give an indication of their function.